src/controller/python/test/test_scripts/base.py

```diff
@@ -179,3 +196,15 @@ def TestWriteBasicAttributes(self, nodeid: int, endpoint: int, group: int):
             self.logger.exception(f"Following attributes failed: {failed_zcl}")
             return False
         return True
+
+    def TestPythonAPI(self):
```

Comment:
please consider changing this def to a more descriptive test name.

`TestPythonAPI` is very generic in this case

Reply:
I would expect this function will contain more "stateless" API tests for CSG use. Renamed it to TestNonControllerAPIs